### PR TITLE
added intial content for readthedocs documentation

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- Added initial `readthedocs` content.
- Configured `readtheorg` and added webhook to build doc.

https://cephci.readthedocs.io/en/latest/index.html#

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "CEPH-CI"
+copyright = "2021, RHCS-QE"
```
Comment (psathyan) [inline docs/conf.py:27]: IMO, it should be `copyright = '2021, Red Hat Inc'`

File: docs/source/contribution_guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
+Contribution Guidelines
+***********************
+
+Being open, we expect contributions from various walks of life. The below guidelines help in keeping the code base uniform irrespective of the person contributing to it.
```
Comment (psathyan) [inline docs/source/contribution_guidelines.rst:4]: With this project being open source, it is important to have a uniform standard. The following guidelines will helps to have a uniform style, being it design or visual`.

File: docs/source/contribution_guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+Coding
+------
+* Follow the PEP 8 proposal. Specifically, for the below mentioned areas
+    - `Naming Conventions`_
+    - `Programming recommendations`_
+    - `Whitespaces`_
+    - Using code comments sparsely but if required then `PEP8`_
+
+* Docstrings is mandatory, it needs to be compact and informative. It is required for each level i.e. functions, methods, classes and modules.
+    - Follow the guidelines from `PEP257`_
```
Comment (psathyan) [inline docs/source/contribution_guidelines.rst:15]: We could reword ` Follow the guidelines as stated in PEP257`

File: docs/source/contribution_guidelines.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+Testing
+-------
+- Unit test the changes
+- Run system tests if required
+- Ensure to run tox before commit
+
+Best Practices
+--------------
+Let’s apply some of the industry standard practices during our development
```
Comment (psathyan) [inline docs/source/contribution_guidelines.rst:54]: Over here, the following best practices for development have been chosen from the vast set of them.

**Review by psathyan — COMMENTED** [review]
Thanks for this PR. It consolidates a lot of things...

Some minor comments.

**Review by psathyan — APPROVED** [review]
Let's get it merged. We can refine as we move forward based on feedback and discussions.

**Comment by sunilkumarn417** [comment]
> Let's get it merged. We can refine as we move forward based on feedback and discussions.

@psathyan , completely missed on updating comments. 
Sure, We can plan and classify the items in the documentation.

let me make changes w.r.t `repo` and `branch` in `readtheorgs` after merge.

**Comment by rakeshgm** [comment]
@sunilkumarn417 looks nice. I like it.

**Pull request merged**